bindings/python/CMakeLists.txt

```diff
@@ -44,6 +37,11 @@ SET (CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/tools/cmake")
 
 CMAKE_POLICY (SET "CMP0048" NEW)
 
+### Options
+
+# Add a subdirectory with a CMakeLists.txt file specifically for wheels generation ?
+OPTION (BUILD_WHEELS_IN_CMAKE "Build python wheels with cmake." ON)
```

Comment:
This option allows turning off python wheels generation in the CMake process, hence letting the CI taking care of the proper wheels generation in the future (using `cibuildwheel` for instance)